packages/public-api/.env.example

```
@@ -21,6 +21,9 @@ UNCHAINED_GNOSIS_HTTP_URL=https://dev-api.gnosis.shapeshift.com
 UNCHAINED_DOGECOIN_HTTP_URL=https://dev-api.dogecoin.shapeshift.com
 UNCHAINED_LITECOIN_HTTP_URL=https://dev-api.litecoin.shapeshift.com
 UNCHAINED_BITCOINCASH_HTTP_URL=https://dev-api.bitcoincash.shapeshift.com
+UNCHAINED_ZCASH_HTTP_URL=https://dev-api.zcash.shapeshift.com
+UNCHAINED_SOLANA_HTTP_URL=https://dev-api.solana.shapeshift.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UNCHAINED_SOLANA_HTTP_URL key should go before the UNCHAINED_THORCHAIN_HTTP_URL key

(UnorderedKey)